dref/imminent_utils.py

```diff
@@ -107,61 +104,54 @@ def parse_type_of_onset(type):
     return type
```

Comment:
Shadows built-in name 'type' – could we use some other variable name here? E.g. onset_type?